### DISCUSSION TITLE
简体中文不翻译

### DISCUSSION THREAD

**Discussion by ling-yuik** [he3-app/he3-feedback#168] [Bug Report]
v1.3.2
简体中文不翻译
![image](https://user-images.githubusercontent.com/65401151/236395578-1791a8e0-a035-42ae-b2fd-afa5d9b67b82.png)
其他语言正常
![image](https://user-images.githubusercontent.com/65401151/236395700-68af87b1-a527-4180-9d89-99995f98a0cf.png)
![image](https://user-images.githubusercontent.com/65401151/236395864-aee76ad6-bdaa-49c2-8d73-6bb6506397da.png)

**Comment by lyzhang1999** [comment]
感谢您的建议，我已将问题反馈给产品部门，您可以关注该 issues：https://github.com/he3-app/he3-feedback/issues/169
